cranelift/filetests/Cargo.toml

```diff
@@ -36,3 +36,7 @@ cranelift-wasm.workspace = true
 wasmparser.workspace = true
 cranelift.workspace = true
 expect-test = "1.4.1"
+
+[dev-dependencies]
+env_logger = "0.10"
```

Comment:
Use `.workspace` dependencies instead. While at it, if you could update `expect-test` do be the same, that would be great.

Reply:
Hmm, I don't fully understand how it should be

Reply:
See the dependencies above such as `wasmparser.workspace = true`. For these dependencies the actual versions and such are maintained in a single place in the `Cargo.toml` file at the root of the project.

These should be the same.

Reply:
Fixed [here](https://github.com/near/wasmtime/pull/40/commits/c13f5989ca1b6366e4106008333efe3baf1e0cc5)